portfolio/src/main/java/com/google/sps/data/Task.java

```diff
@@ -0,0 +1,39 @@
+package com.google.sps.data;
+
+/** Models a task that the user creates. */
+public final class Task {
+
+  private final String name;
+  private final String description;
+  private final Integer durationMinute;
+  private final Integer priority;
```

Comment:
I would suggest that you introduce a TaskPriority class. That way, for example, you can enforce range.

Reply:
Done.